HW05.Rmd

```diff
@@ -130,13 +139,23 @@ CTRP_df_q2 <- CTRP_df %>%
   filter(cancer_type == "prostate", ccl_name == "22RV1") %>%
   group_by(cpd_name) %>%
   summarize(AUC = mean(area_under_curve)) %>%
-  arrange(AUC)
-
+  arrange(AUC) %>%
+  top_n(-10, AUC) #I cheat a little bit in here. Because there are so many compounds that interact with 22RV1 that in the graph it becomes difficult to read the labels, I decided to narrow the number of compounds to the top (or "bottom" in this case, hence the negative number) 10 with the lowest AUC. 
```

Comment:
I wouldn't call this cheating! It seems like a great way to get only the data you want/need without excess. Also a great demonstration of a function we learned from the datacamp lectures